packaging/rpm-ostree.spec.in

```diff
@@ -135,17 +135,14 @@ The %{name}-devel package includes the header files for %{name}-libs.
 %autosetup -Sgit -n %{name}-%{version}
 
 %build
-# gobject introspection does not work with LTO.  There is an effort to fix this
-# in the appropriate project upstreams, so hopefully LTO can be enabled someday
-# Disable LTO.
-%define _lto_cflags %{nil}
-
 env NOCONFIGURE=1 ./autogen.sh
 # Since we're hybrid C++/Rust we need to propagate this manually;
 # the %%configure macro today assumes (reasonably) that one is building
 # C/C++ and sets C{,XX}FLAGS
+%if 0%{?build_rustflags:1}
 export RUSTFLAGS="%{build_rustflags}"
-%configure --disable-silent-rules --enable-gtk-doc %{?sqlite_rpmdb_default} %{?with_sanitizers:--enable-sanitizers} 
+%endif
+%configure --disable-silent-rules --enable-gtk-doc %{?sqlite_rpmdb_default} %{?with_sanitizers:--enable-sanitizers}
```

Comment:
Aside: I was so confused here why it was saying this line changed and thought it was some kind of degenerate case in the diff algorithm. But then I saw the trailing whitespace. :)